tictactoe.py

```diff
@@ -2,8 +2,7 @@
 
 BOARD_SIZE = 3
 
-class board:
-
+class Board:
```

Comment:
PEP8 shows this should be capitalized:  https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments